### PR TITLE
Project Tests

### PR DESCRIPTION
**Pull request by rickonnen**
Add project tests
Delete project tests
Update project tests

Worked By Giovanny Uscamaita

Notes:
I may have questions for a method that let us click the Create project/workspace button, sometimes works, other not.

### CODE REVIEW

File: src/main/java/org/fundacionjala/pivotal/framework/util/CommonMethods.java [hunk 1]
```diff
@@ -2,18 +2,21 @@
 
 import java.util.ArrayList;
 import java.util.Map;
+import java.util.NoSuchElementException;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/framework/util/CommonMethods.java]: please use the NoSuchElementException from selenium

File: src/main/java/org/fundacionjala/pivotal/pages/BasePage.java [hunk 2]
```diff
@@ -18,15 +18,15 @@
...
+    //protected Actions action;
 
     /**
      * Class constructor.
      */
     public BasePage() {
         driver = DriverManager.getInstance().getDriver();
         wait = new WebDriverWait(driver, WAIT_TIME);
-        action = new Actions(driver);
+        //action = new Actions(driver);
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/pages/BasePage.java]: Remove commented code

**Comment by carledriss** [comment]
@rickonnen 
Please fix checkstyle issues.

**Comment by rickonnen** [comment]
Changes referred Carlos observations inserted,
updated some methods used inside the test methods.

**Comment by carledriss** [comment]
LGTM ! :+1:

**Pull request merged**